### PR TITLE
Feature: allow user to change editor theme

### PR DESCRIPTION
**Pull request by ChiragKParmar**
Adding theme for editor: screen shots below
theme5
theme4
theme3
theme2

### CODE REVIEW

**Comment by ChiragKParmar** [comment]
Adding theme for editor: screen shots below
theme2
theme5
theme4
theme3

File: rd_ui/app/embed.html [hunk 1]
```diff
@@ -12,6 +12,12 @@
     <!-- build:css /styles/embed.css -->
     <link rel="stylesheet" href="/styles/superflat_redash.css">
     <link rel="stylesheet" href="/bower_components/codemirror/lib/codemirror.css">
+    <link rel="stylesheet" href="/bower_components/codemirror/theme/3024-day.css">
+    <link rel="stylesheet" href="/bower_components/codemirror/theme/3024-night.css">
+    <link rel="stylesheet" href="/bower_components/codemirror/theme/eclipse.css">
+    <link rel="stylesheet" href="/bower_components/codemirror/theme/monokai.css">
+    <link rel="stylesheet" href="/bower_components/codemirror/theme/neat.css">
+    <link rel="stylesheet" href="/bower_components/codemirror/theme/elegant.css">
```
Comment (arikfr) [inline rd_ui/app/embed.html]: No need to add this in `embed.html` as we don't use the editor in embeds.

**Comment by arikfr** [comment]
This is cool, but I don't feel it's worth making the UI more crowded for this feature. Also without persisting the option, it will be somewhat frustrating for the user to keep selecting the theme over and over again.

Maybe we can move this to user configuration? Or "hide" under some menu in the query screen (although still need to add persistence to the option).

**Comment by ChiragKParmar** [comment]
I guess you are right! I like the option of moving to some menu in the query screen. And agree that unless we persist users choice, this adds little value. Let me work on that part. You can either decline this pull request or I will make changes on the same. Thanks

**Comment by arikfr** [comment]
OK, I will wait for the updated version.

**Comment by arikfr** [comment]
Please re-open when relevant. Thanks!

**Pull request closed without merging**